ods_ci/tests/Resources/Files/accelerator/qe_create_ap_2.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  generation: 1
+  managedFields:
```

Comment:
Do we need to have these `managedFields` in all these files?

Reply:
No, we don't really need them. Thanks for pointing it!
Fixed in [ded2638](https://github.com/red-hat-data-services/ods-ci/pull/1209/commits/ded2638dc617dfd07cdd153efcb3d31a485ee7b7)